sql-indent.el

```diff
@@ -1752,7 +1755,10 @@ This is an indentation adjuster and needs to be added to the
     ;; consider them an operator and line up the line to the first word of the
     ;; line, not the operator
     (cond ((looking-at "\\W+")
-	   (let ((ofs (length (match-string 0))))
+	   (let* ((operator (match-string 0))
+		  (ofs (length operator)))
+	     (when (string-match "'$" operator)
```

Comment:
This is not going to work if the string itself starts with a non word, like this:

```sql
select c
        || '++'
from my_table
```

It seems to me that the safest way is to change the `looking-at` regexp to list the operators explicitly.  SQLite seems to have the [following](https://sqlite.org/lang_expr.html) operators (I suspect Oracle has pretty much the same.  I think we can build a regexp with `regexp-opt` that just lists them.  Something like:

```emacs-lisp
(defconst sqlind-operator-regexp
  (concat (regexp-opt '("||" "*" "/" "%" "+" "-" "<<" ">>" "&" "|"
                        "<" "<=" ">" ">=" "=" "==" "!=" "<>")
                      t)
          "\\s-+"))
```

This way, we only match what we need and don't have to undo the extra match.